src/main/java/mops/gruppen2/domain/model/group/wrapper/Parent.java

```diff
@@ -0,0 +1,36 @@
+package mops.gruppen2.domain.model.group.wrapper;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.ToString;
+import lombok.Value;
+import mops.gruppen2.domain.helper.CommonHelper;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.beans.ConstructorProperties;
+import java.util.UUID;
+
+@Value
+@ToString
+public class Parent {
+
+    @NotNull
+    @JsonProperty("id")
+    UUID value;
+
+    @ConstructorProperties("id")
+    public Parent(@NotBlank @Size(min = 36, max = 36) String parentid) {
+        value = UUID.fromString(parentid);
+    }
+
+    public static Parent EMPTY() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The static method name 'EMPTY' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/tobi_2/gruppenfindung/pullRequest?prid=5356872)